### PR TITLE
Add boolean modifiers for Adventure Randomly, Alters Page Text, and Blind

### PR DESCRIPTION
**Pull request by Rinn**
Add boolean modifiers for Adventure Randomly, Alters Page Text, and Blind

```
> jsl Effect.all().filter(x => booleanModifier(x, "Blind")).map(x => x.name)

Returned: aggregate string [1]
0 => Temporary Blindness

> jsl Item.all().filter(x => booleanModifier(x, "Blind")).map(x => x.name)

Returned: aggregate string [4]
0 => makeshift turban
1 => sleep mask
2 => Helps-You-Sleep
3 => Amulet of Perpetual Darkness

> jsl Effect.all().filter(x => booleanModifier(x, "Adventure Randomly")).map(x => x.name)

Returned: aggregate string [3]
0 => Teleportitis
1 => Funday!
2 => Feeling Lost

> jsl Item.all().filter(x => booleanModifier(x, "Adventure Randomly")).map(x => x.name)

Returned: aggregate string [1]
0 => ring of teleportation

> jsl Effect.all().filter(x => booleanModifier(x, "Alters Page Text")).map(x => x.name)

Returned: aggregate string [6]
0 => Can Has Cyborger
1 => O Hai!
2 => Haiku State of Mind
3 => Dis Abled
4 => Just the Best Anapests
5 => Robocamo

> jsl Item.all().filter(x => booleanModifier(x, "Alters Page Text")).map(x => x.name)

Returned: aggregate string [2]
0 => sword behind inappropriate prepositions
1 => staph of homophones
```

### CODE REVIEW

**Comment by Rinn** [comment]
I think I got all the effects and equipment for these, lmk if I missed anything or if there are other modifiers we should track

**Comment by midgleyc** [comment]
For the page text modifiers:
* I don't think Dis Abled turns the text to anapests?
* metallic foil cat ears work the same way as O Hai!
* robocamo is in there because it replaces the player name?
* hewn moon rune spoon and the replica version both replace combat text, though Mafia corrects this (or it's in the title text in game)
* papier-mache equipment also replace combat text: papier-mitre, papier-mâchuridars, papier-mâchéte, papier-mâchine gun, papier-masque, and Mafia does not correct this (and it's not in title text)

**Comment by gausie** [comment]
I gave this feedback to you elsewhere but April Fools as a holiday should also impart Alters Page Text

**Comment by Rinn** [comment]
Okay did all this. Are we a yes or no on robocamo because it changes the player name

File: src/net/sourceforge/kolmafia/KoLCharacter.java [hunk 1]
```diff
@@ -5112,6 +5112,9 @@ public static final Modifiers recalculateAdjustments(
           DoubleModifier.RESTING_HP_PCT, 100, ModifierType.EVENT, "Moons (Grimace full)");
     }
 
+    // Words are replaced with salad on April 1st (hilarious)
+    newModifiers.setBoolean(BooleanModifier.ALTERS_PAGE_TEXT, HolidayDatabase.isAprilFirst());
```
Comment (midgleyc) [inline src/net/sourceforge/kolmafia/KoLCharacter.java:5116]: I think this can go into `modifiers.txt`:
```
Event	April Fool's Day	Alters Page Text
```

**Comment by gausie** [comment]
> Okay did all this. Are we a yes or no on robocamo because it changes the player name

I would say no - it won't affect scripting. It only changes the name on the profile page right?

**Comment by Rinn** [comment]
probably not

**Comment by Rinn** [comment]
this should be ready to go

**Comment by midgleyc** [comment]
The modifiers.txt change for April Fools' Day doesn't seem to have made it in.

**Comment by Rinn** [comment]
Ah missed that, done

**Review by midgleyc — APPROVED** [review]
Thanks!

**Pull request merged**